README.md

```diff
@@ -2,139 +2,140 @@
 
 # Nyaml to nxdl
 
-A tool to convert yaml NeXus application definitions (nyaml) to nexus definitions language (nxdl) and vice versa.
+Nyaml to Nxdl is a versatile tool designed for converting NeXus application definitions from YAML (nyaml) to the Nexus Definitions Language (nxdl) format and vice versa. This command-line interface offers a simple and efficient way to work with NeXus data definitions.
```

Comment:
Do we want to say **Nxdl, Nyaml** or **NXDL, Nyaml**? I suggest the later to have the same notation as YAML and XML.

Reply:
Yes, I agree that it's best to call the tool itself `nyaml`